### ISSUE TITLE
a problem loading the pre training model

### ISSUE THREAD

**Issue by whenaway** [YueLiao/PPDM#64]
Hello, I encountered some problems in training while learning your code. 
When the pre training model is loaded, no param is displayed  
The error log is as follows.I hope to receive your reply as soon as possible.
python main.py  Hoidet --batch_size 112 --master_batch 7 --lr 4.5e-4 --gpus 0,1,2,3,4,5,6,7  --num_workers 16  --load_model ../models/ctdet_coco_dla_2x.pth --image_dir images/train2015 --dataset hico --exp_id hoidet_hico_dla
Fix size testing.
training chunk_sizes: [7, 15, 15, 15, 15, 15, 15, 15]
The output will be saved to  /home/b324/WZF/PPDM-master/src/lib/../../exp/Hoidet/hoidet_hico_dla
heads {'hm': 80, 'wh': 2, 'hm_rel': 117, 'sub_offset': 2, 'obj_offset': 2, 'reg': 2}
Namespace(K=100, arch='dla_34', batch_size=112, cat_spec_wh=False, chunk_sizes=[7, 15, 15, 15, 15, 15, 15, 15], data_dir='/home/b324/WZF/PPDM-master/src/lib/../../data', dataset='hico', debug=0, debug_dir='/home/b324/WZF/PPDM-master/src/lib/../../exp/Hoidet/hoidet_hico_dla/debug', debugger_theme='white', demo='', dense_wh=False, down_ratio=4, exp_dir='/home/b324/WZF/PPDM-master/src/lib/../../exp/Hoidet', exp_id='hoidet_hico_dla', fix_res=True, flip=0.5, flip_test=False, gpus=[0, 1, 2, 3, 4, 5, 6, 7], gpus_str='0,1,2,3,4,5,6,7', head_conv=256, heads={'hm': 80, 'wh': 2, 'hm_rel': 117, 'sub_offset': 2, 'obj_offset': 2, 'reg': 2}, hide_data_time=False, hm_weight=1, image_dir='images/train2015', input_h=512, input_res=512, input_w=512, keep_res=False, load_model='../models/ctdet_coco_dla_2x.pth', lr=0.00045, lr_step=[90, 120], master_batch_size=7, mean=array([[[0.40789655, 0.44719303, 0.47026116]]], dtype=float32), metric='loss', mse_loss=False, nms=False, no_color_aug=False, norm_wh=False, not_cuda_benchmark=False, not_prefetch_test=False, not_rand_crop=False, not_reg_offset=False, num_classes=80, num_classes_verb=117, num_epochs=140, num_iters=-1, num_stacks=1, num_workers=16, off_weight=1, output_h=128, output_res=128, output_w=128, pad=31, print_iter=0, reg_loss='l1', reg_offset=True, resume=False, root_dir='/home/b324/WZF/PPDM-master/src/lib/../..', root_path='../Dataset', rotate=0, save_all=False, save_dir='/home/b324/WZF/PPDM-master/src/lib/../../exp/Hoidet/hoidet_hico_dla', save_predictions=False, save_video='', scale=0.4, seed=317, shift=0.1, std=array([[[0.2886383 , 0.27408165, 0.27809834]]], dtype=float32), task='Hoidet', test=False, test_dir='', test_scales=[1.0], test_video=False, test_with_eval=False, trainval=False, use_cos=0, use_verb_sub=0, val_intervals=100000, vis_thresh=0.3, wh_weight=0.1)
Creating model...
loaded ../models/ctdet_coco_dla_2x.pth, epoch 230
No param hm_rel.0.weight.
No param hm_rel.0.bias.
No param hm_rel.2.weight.
No param hm_rel.2.bias.
No param sub_offset.0.weight.
No param sub_offset.0.bias.
No param sub_offset.2.weight.
No param sub_offset.2.bias.
No param obj_offset.0.weight.
No param obj_offset.0.bias.
No param obj_offset.2.weight.
No param obj_offset.2.bias.
